scripts/build.py

```diff
@@ -77,8 +77,12 @@
 MANIFEST_FILE_PATH = os.path.join('manifest.json')
 
 REMOVE_WS = re.compile(r'\s{2,}').sub
-YUICOMPRESSOR_DIR = os.path.join(
-    '..', 'oppia_tools', 'yuicompressor-2.4.8', 'yuicompressor-2.4.8.jar')
+
+# This path is only used for java command line. But the Windows style path
+# is causing syntax error. Since Unix style path can also work, we are
+# safe to just use this style.
+YUICOMPRESSOR_DIR = '/'.join([
```

Comment:
Perhaps use os.path.sep, os.pardir?

Reply:
you have the latter, not the former :)

Reply:
`os.path.sep` is `\` on Windows, which may cause java's syntax error. But I will update the `os.pardir` one.

Reply:
ah, that makes more sense now.